cmd/main.go

```diff
@@ -744,158 +452,448 @@ func Run() error {
 		}
 
 		return InitCommand(config)
+	}
+
+	// ================================
+	// @todo are invites blocks?
```

Comment:
Yep, they are, but they do not stay on the update chain / graph and inbound invites get indexed into the `invites` table